### PR TITLE
Home feature

### PR DESCRIPTION
**Pull request by lucasscurtoo**
adding home user, movies and promises to the API

### CODE REVIEW

File: src/pages/main.jsx [hunk 1]
```diff
@@ -0,0 +1,60 @@
+import '../App.css';
```
Comment (naipeloz) [inline src/pages/main.jsx:1]: Same as above

File: src/pages/main.jsx [hunk 2]
```diff
@@ -0,0 +1,60 @@
...
+import HomePrices from '../components/homePrice'
+import HomeCancel from '../components/homeCancel';
+import HomeWatch from '../components/homeWatch';
+import Footer from '../components/footer';
+import Header from '../components/header'
+
+function Main(){
+    let [homeState, setHomeState] = useState('price');
+
+    function cargarSeccion(){
```
Comment (naipeloz) [inline src/pages/main.jsx:13]: Why not use a switch instead of the for?

File: src/pages/userHome.jsx [hunk 3]
```diff
@@ -0,0 +1,109 @@
+
+/* eslint-disable no-undef */
```
Comment (naipeloz) [inline src/pages/userHome.jsx:2]: Why did you exclude these rules?

File: src/styles/carouselStyles.css [hunk 4]
```diff
@@ -0,0 +1,39 @@
+
+.carousel-img{
+    width: fit-content;
+    height: fit-content;
+    display: flex;
+    align-items: center;
+    
+
```
Comment (naipeloz) [inline src/styles/carouselStyles.css:8]: Leave just one blank line

File: src/styles/userHome.css [hunk 5]
```diff
@@ -0,0 +1,73 @@
+*{
```
Comment (naipeloz) [inline src/styles/userHome.css:1]: Same as above

File: src/utilities/functions.js [hunk 6]
```diff
@@ -0,0 +1,4 @@
+/* eslint-disable no-unreachable */
+ export let getRandomNumber = function (max) {
+    return Math.floor(Math.random() * max );
+}
```
Comment (naipeloz) [inline src/utilities/functions.js:4]: Add a blank line

**Review by naipeloz — CHANGES_REQUESTED** [review]

File: craco.config.js [hunk 7]
```diff
@@ -0,0 +1,10 @@
+module.exports = {
+    style: {
+      postcss: {
+        plugins: [
+          require('tailwindcss'),
+          require('autoprefixer'),
+        ],
+      },
+    },
+  }
```
[the remark this answers is not in the document]
Comment (lucasscurtoo) [reply craco.config.js:10]: didnt understand :(

File: src/pages/main.jsx (hunk 2 above)
Comment (lucasscurtoo) [reply src/pages/main.jsx:13]: did you mean a switch instead of the if's?

**Pull request merged**